lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -130,4 +186,27 @@ def parse_gemfile_lock
       Bundler::LockfileParser.new(gemfile_contents)
     end
   end
+
+  def major_bundler_version
+    # https://rubular.com/r/uuRpai9IheL68d
+    bundler_version_match = @gemfile_lock_path.read.match(/^BUNDLED WITH$(\r?\n)   (?<major>\d*)\.\d*\.\d*/m)
+
+    if bundler_version_match
+      bundler_version_match[:major]
+    else
+      "1"
```

Comment:
Should we actually fail instead of defaulting?

Reply:
People can still deploy without having a `BUNDLED WITH` in their Gemfile.lock. For example in some of our hatchet test apps https://github.com/sharpstone/default_ruby/blob/master/Gemfile.lock.